data-entry/dataEntry.app.js

```diff
@@ -78,12 +78,62 @@
                         if (key.simple) {
                             (function(key) {
                                 key.getDomainValues().then(function success(values) {
-                                    dataEntryModel.domainValues[key.colset.columns[0].name] = [];
-                                    var domainValues = dataEntryModel.domainValues[key.colset.columns[0].name];
-
-                                    angular.forEach(values.data, function(value) {
-                                        var field = Object.keys(value)[0];
-                                        domainValues.push(value[field]);
+                                    var domainValues = dataEntryModel.domainValues[key.colset.columns[0].name] = [];
+                                    var table = key.key._table;
```

Comment:
`_table` is a private variable so it should not be used. Is the ermrestjs documentation confusing here? Let us know.